dev-tools/mage/settings.go

```diff
@@ -19,11 +20,13 @@ import (
 
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
+	"gopkg.in/yaml.v3"
 
 	"github.com/magefile/mage/sh"
 	"golang.org/x/tools/go/vcs"
```

Comment:
Created dedicated issue for vcs removal https://github.com/elastic/elastic-agent/issues/4138